### PR TITLE
Support added for negative starting or ending balance

### PR DESCRIPTION
**Pull request by tswestendorp**
It looked like the starting and ending balance were always assumed to be a positive.

This pr adds support for negative balances

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 composer.lock
+/.idea/
```
Comment (fruitl00p) [inline .gitignore]: I think all specific IDE stuff (like .idea directory created by JetBrains' IDE shouldnt be here...) You should move that over to your local global .gitignore file... 

(see for example here http://stackoverflow.com/questions/6626136/best-practice-for-adding-gitignore-to-repo)

**Comment by fruitl00p** [comment]
Cool! Looks good to me, although i'd like to see the gitignore change reverted since i dont believe that should be in the project itself, but in your [local global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

**Comment by tswestendorp** [comment]
No problem :) 

Reverted that commit now

**Comment by fruitl00p** [comment]
LGTM :+1:

**Comment by fruitl00p** [comment]
Thanks!

**Pull request merged**